### ISSUE TITLE
1.13 blog post: Add breaking change message for observable.set

### ISSUE THREAD

**Issue by mixonic** [emberjs/website#2212]
re: https://github.com/emberjs/ember.js/pull/11213

**Comment by slindberg** [comment]
Just got burned by the return value of `Ember.setProperties`, and didn't remember seeing it in the breaking changes.

Is [this](https://github.com/emberjs/website/blob/master/source/blog/2015-08-13-ember-2-0-released.md) the blog post you wanted to put it into? Is an old blog post the best place for this now anyway?

**Comment by acorncom** [comment]
@locks @mixonic Thoughts on this?  Do we still note this somewhere?

**Comment by Serabe** [comment]
I was actually asked about the motivation behind this in the last Ember Madrid Meetup.

**Comment by mixonic** [comment]
Yeah, it still seems good to add an item into the blog post if someone can pick it up, and possibly in the deprecation guide for 1.13? In the blog post it could go at the end of "Removed APIs".

It was listed in the Changelog. It is unfortunate that it was not included in the post.

**Comment by jessica-jordan** [comment]
Should this be included into the 1.13 blog post or rather into the 2.0 blog post as a deprecation? I saw that there already was an open PR for the deprecations Guide [here](https://github.com/emberjs/website/pull/2315/files) which referenced to the 2.0 release instead. What do you think @mixonic?

If this seems valid, I'd like to open another issue for this with additional information so it might be possible to be picked up by a first-time contributor as well
